tests/test_cloze.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+"""Test creating Cloze cards"""
+
+from genanki import Model
+from genanki import Note
+from genanki import Deck
+from genanki import Package
+
+CSS = """.card {
+ font-family: arial;
+ font-size: 20px;
+ text-align: center;
+ color: black;
+ background-color: white;
+}
+
+.cloze {
+ font-weight: bold;
+ color: blue;
+}
+.nightMode .cloze {
+ color: lightblue;
+}
+"""
+
+MY_CLOZE_MODEL = Model(
+  998877661,
+  'My Cloze Model',
+  fields=[
+    {'name': 'Text'},
+    {'name': 'Extra'},
+  ],
+  templates=[{
+    'name': 'My Cloze Card',
+    'qfmt': '{{cloze:Text}}',
+    'afmt': '{{cloze:Text}}<br>{{Extra}}',
+  },],
+  css=CSS,
+  model_type=1)
+
+def test_cloze():
+  """Test Cloze model"""
+  notes = []
+  assert MY_CLOZE_MODEL.to_json(0, 0)["type"] == 1
+
+  fields = ['NOTE ONE: {{c1::single deletion}}', '']
+  my_cloze_note = Note(model=MY_CLOZE_MODEL, fields=fields)
+  assert {card.ord for card in my_cloze_note.cards} == {0}
+  notes.append(my_cloze_note)
+
+  fields = ['NOTE TWO: {{c1::1st deletion}} {{c2::2nd deletion}} {{c3::3rd deletion}}', '']
+  my_cloze_note = Note(model=MY_CLOZE_MODEL, fields=fields)
+  assert {card.ord for card in my_cloze_note.cards} == {0, 1, 2}
```

Comment:
Probably better to assert the sorted list is equal, i.e.

```
assert sorted(card.ord for card in my_cloze_note.cards) == [0, 1, 2]
```

Otherwise there could be duplicate `ord` values in `my_cloze_note.cards` and this test wouldn't catch them.

Reply:
Missed that. Thanks. This is addressed with: https://github.com/kerrickstaley/genanki/pull/37/commits/c61a8bc901f01a205b0888b17afa47e01f5aaeca